tests/test_project/tests/test_qa.py

```diff
@@ -6,13 +6,16 @@
 # Mock is a standard library from python3.3-pre onwards
 # from unittest.mock import patch
 from mock import patch
-from openwisp_utils.qa import check_migration_name
+from openwisp_utils.qa import check_commit_message, check_migration_name
 
 MIGRATIONS_DIR = path.join(path.dirname(path.dirname(path.abspath(__file__))), 'migrations')
 
 
 class TestQa(TestCase):
     _test_migration_file = '%s/0002_auto_20181001_0421.py' % MIGRATIONS_DIR
+    _gh_token = os.environ["GH_TOKEN"]
+    _repo = os.environ["TRAVIS_REPO_SLUG"]
+    _commit = os.environ["TRAVIS_COMMIT"]
```

Comment:
I don't think you need to do this here, you need to supply your own test values directly